### PR TITLE
Code Reviewer

### PR DESCRIPTION
**Pull request by kamranasif506**
Key features are 

- [ ] copied code from previous work

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8"> 
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (kamranasif506) [inline index.html:6]: It is great that you give a title to your page but there should be some modification needed. Please give the title according to the page. As the design of the page is a survey form and the heading is also a "A survey form" so it is better that the title of the page should be according to this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          <option>Excellent</option>
+          <option>Very Good</option>
+          <option>Good</option>
+        </select>
+        <label>How do you rate our facilities</label>
+        <select class="dropdown">
+          <option>Excellent</option>
+          <option>Very Good</option>
+          <option>Good</option>
+        </select>
```
Comment (kamranasif506) [inline index.html:37]: Please don't leave the <option> tag values. you should assign each <option> tag value in order to send data. here is an example`<option value="Excellent">Excellent</option>`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (kamranasif506) [inline index.html:56]: Please remove this comment lines as this is not a good approach to leave unused comment lines in the codebase due to this your code will not be considered as clean.

**Review by kamranasif506 — COMMENTED** [review]
Good job so far!
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪


### Highlights
✅ Professional and custmozide README.
✅ Linters tests passing.
✅ Correct use of GitHub Flow.

# Changes Required ♻️ 
Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by pinkmoon25 — APPROVED** [review]
Hi @kamranasif506 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 

## Optional suggestions
**[OPTIONAL] In the highlights of your review you mentioned "Professional and custmozide README" but there is no Readme so I would suggest changing it accordingly.** 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**